sequencer/src/block2.rs

```diff
@@ -0,0 +1,144 @@
+// Question: jellyfish uses `ark_std` everywhere instead of `std`.
+// Is there a chance of badness if we (as a downstream user) use `std`?
```

Comment:
I don't think so, with the `std` feature enabled `ark_std` should just be a wrapper around `std`